### DISCUSSION TITLE
Why containers will lost after computer shutdown?

### DISCUSSION THREAD

**Discussion by wxqsly** [rancher-sandbox/rancher-desktop#3419] [Q&A] [answered]
Hi, 
I am using Rancher Desktop 1.6.2 on windows for docker container management just one week. 
However I found that each time I shut down my computer where Rancher Desktop installed, the next time I start computer and access to Rancher Desktop, all the containers previously ran are gone.
The 'gone' meaning not stopped, but rather nothing if i ran command 'docker ps', then ther e is no single container list (or empty)

`docker ps

CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES`

would like to know, is there something wrong with my configuration? How should I use it in correct way?
Or is it intended for Rancher Desktop?

thanks.

**Comment by wxqsly** [comment, marked as answer]
fixed.
I misunderstood this problem.
actually the constainers are there but just hidden by the 'docker ps' command.
I got the clue from https://github.com/rancher-sandbox/rancher-desktop/discussions/3225
and now i can see it with the correct command.

**Comment by jandubois** [comment]
If you want your containers to start up again automatically, you have to create them with the ` --restart=always` option, i.e.

```
docker run -d -p 8080:80 --restart=always nginx
```
